modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/exportimport/portlet/preferences/processor/DDMFormExportImportPortletPreferencesProcessor.java

```diff
@@ -90,6 +93,39 @@ public PortletPreferences processExportPortletPreferences(
 			return portletPreferences;
 		}
 
+		long groupId = GetterUtil.getLong(
+			portletPreferences.getValue("groupId", null));
+
+		if (groupId <= 0) {
```

Comment:
Hi @vendeltoreki, is there any special reason for you check if the `groupId` is a negative number? From what I can see, the `groupId` will be a positive number or `0` once the method `GetterUtil.getLong` is being invoked to set the value of this variable.

Reply:
Hi @natocesarrego , I've basically copy-pasted the code from [JournalContentExportImportPortletPreferencesProcessor](https://github.com/liferay/liferay-portal/blob/b3173da81b62430f7150b695d5e944b224279580/modules/apps/journal/journal-content-web/src/main/java/com/liferay/journal/content/web/internal/exportimport/portlet/preferences/processor/JournalContentExportImportPortletPreferencesProcessor.java#L124-L155). I did not find any uses of negative `groupId`s either, so my guess is that we can just check whether `groupId == 0`.

Reply:
Great! Thanks for your clarification. 🙂